website/client-old/js/services/taskServices.js

```diff
@@ -72,10 +72,16 @@ angular.module('habitrpg')
     };
 
     function updateTask (taskId, taskDetails) {
+      var taskDetailsToSend = _.omit(
+        taskDetails,
+        ['challenge', 'completed', 'createdAt', 'group', 'history', 'id',
```

Comment:
Question for the room: Should we just prevent updating complex fields (`challenge`, `group`, `history`, `reminders`, `tags`) and just let the other negligible properties be ignored? The problem is that we're sending too much data along with the request, so if we cut out the non-string stuff, it would make the code simpler.

```js
var taskDetailsToSend = _.omit(taskDetails, ['challenge', 'group', 'history', 'reminders', 'tags']);
```